src/test/addresses_mainnet.sol

```diff
@@ -491,5 +490,25 @@ contract Addresses {
         addr["CRON_ORACLE_JOB"]                  = 0xe717Ec34b2707fc8c226b34be5eae8482d06ED03;
         addr["CRON_FLAP_JOB"]                    = 0xc32506E9bB590971671b649d9B8e18CB6260559F;
         addr["CRON_LITE_PSM_JOB"]                = 0x0C86162ba3E507592fC8282b07cF18c7F902C401;
+        addr["USDS"]                             = 0xdC035D45d973E3EC169d2276DDab16f1e407384F;
+        addr["USDS_IMP"]                         = 0x1923DfeE706A8E78157416C29cBCCFDe7cdF4102;
+        addr["USDS_JOIN"]                        = 0x3C0f895007CA717Aa01c8693e59DF1e8C3777FEB;
+        addr["DAI_USDS"]                         = 0x3225737a9Bbb6473CB4a45b7244ACa2BeFdB276A;
+        addr["SUSDS"]                            = 0xa3931d71877C0E7a3148CB7Eb4463524FEc27fbD;
+        addr["SUSDS_IMP"]                        = 0x4e7991e5C547ce825BdEb665EE14a3274f9F61e0;
+        addr["SKY"]                              = 0x56072C95FAA701256059aa122697B133aDEd9279;
+        addr["MKR_SKY"]                          = 0xBDcFCA946b6CDd965f99a839e4435Bcdc1bc470B;
+        addr["PAIR_DAI_MKR"]                     = 0x517F9dD285e75b599234F7221227339478d0FcC8;
+        addr["PAIR_USDS_SKY"]                    = 0x2621CC0B3F3c079c1Db0E80794AA24976F0b9e3c;
```

Comment:
Nit: we previously called those with `UNIV2` prefix, see `UNIV2DAIETH` and other Univ2 LP pairs defined above

Reply:
Hmm, those where for LP tokens used as collateral. Not sure we want to keep the same pattern here, as it might be misleading.

Reply:
In any case, I've updated the names as you suggested. We can reverse the changes later if we decide so.